components/twitter/actions/add-user-to-list/add-user-to-list.mjs

```diff
@@ -0,0 +1,54 @@
+import twitter from "../../twitter.app.mjs";
+
+export default {
+  key: "twitter-add-user-to-list",
+  name: "Add User To List",
+  description: "Add a member to a list. The authenticated user must own the list to be able to add members to it",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    twitter,
+    list: {
+      propDefinition: [
+        twitter,
+        "listSlug",
+      ],
+    },
+    userId: {
+      propDefinition: [
+        twitter,
+        "userId",
+      ],
+      optional: true,
+    },
+    screenName: {
+      propDefinition: [
+        twitter,
+        "screenName",
+      ],
+      optional: true,
+    },
+  },
+  async run() {
+    const {
+      list,
+      userId,
+      screenName,
+    } = this;
+
+    if (!userId && !screenName) {
+      throw new Error("This action requires either User ID or Screen Name. Please enter one or the other above.");
+    }
+
+    const { screen_name: ownerScreenName } = await this.twitter.verifyCredentials();
+
+    const params = {
+      slug: list,
+      ownerScreenName,
+    };
+    if (userId) params.userId = userId;
+    if (screenName) params.screenName = screenName;
+
+    return this.twitter.addUserToList(params);
```

Comment:
Could you please add a summary here and in all actions, please?
You can find an example [here](https://github.com/PipedreamHQ/pipedream/pull/1865/files#diff-b8ca1aa14581a64059c7fad21406e3c0222e7e3931a28c8e2b8a14a5903edf77)

e.g
```suggestion
    const res = await  this.twitter.addUserToList(params);
    $.export("$summary", "User successfully added to the list");
    return res
```

In this case, you need to return the `await`. Sorry for make you remove it